news/4.36/platform_isv.md

```diff
@@ -5,7 +5,25 @@
 ## Platform Changes
 -->
 
-<!--
 ---
 ## SWT Changes
--->
+
+### Lazy Handle Generation for Images and Fonts (Windows only)
+
+When creating resources like image `Image` and `Font`, the necessary OS handles used to be created eager upon instantiation of the resource.
+On Windows, this behavior was changed for images and fonts.
+Handles are now generated upon first access to it.
+Until then, only the metadata to generate the handles is stored.
+
+This is particularly relevant on Windows because of the introduced [monitor-specific scaling](platform.md#monitor-specific-ui-scaling-as-default-windows-only).
```

Comment:
```suggestion
This is particularly relevant on Windows because of the introduction of [monitor-specific scaling](platform.md#monitor-specific-ui-scaling-as-default-windows-only).
```